Makefile

```diff
@@ -0,0 +1,14 @@
+# Format:
+# MAJOR: This is "1" for now. Don't change it.
+# MINOR: This is the current version of the portal API in YYYYMMDD format. Consumers (connlib, REST) will request
+#        this API from the portal with the X-Firezone-API-Version request header.
+# PATCH: Increment this each time you want to publish a new Firezone version.
+version = 1.20231001.0
+
+version:
```

Comment:
I think technically, this should be a `.PHONY` target because you are not actually depending on files that are stored on disk.

Reply:
Good catch! Looks like it got automerged though. I'll fix it now.